base/boot.jl

```diff
@@ -271,6 +271,8 @@ macro nospecialize(x)
 end
 Expr(@nospecialize args...) = _expr(args...)
 
+macro latestworld() Expr(:latestworld) end
+
```

Comment:
It would be highly unsound and unsafe for this to appear anywhere in user code, so I don't know why we should have a macro and AST form of this IR effect. It is only something that should appear post-lowering in toplevel code (illegal in AST forms prelowering).

Also, eventually we need to add this to the verifier, so toplevel code doesn't cause that to error in debug mode

Reply:
It's a lowering error to but this anywhere but toplevel code - I don't see the unsoundness.

Reply:
Ah, okay, yes that seems okay then. I guess I can see why you would want some macros to produce this in some circumstances (although `:toplevel` may be preferred, sometimes it wants to use local variables that prevent using that form)